Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -45,6 +38,10 @@ export async function installCertInTemporaryKeychain(keychainPath: string, keych
     importP12Command.arg(['import', p12CertPath, '-P', p12Pwd, '-A', '-t', 'cert', '-f', 'pkcs12', '-k', keychainPath]);
     await importP12Command.exec();
 
+    //set the partition_id ACL for the private key we just imported so codesign won't prompt to use the key for signing. See
+    //https://stackoverflow.com/questions/39868578/security-codesign-in-sierra-keychain-ignores-access-control-settings-and-ui-p
+    await setKeyPartitionList(keychainPath, keychainPwd, privateKeyName);
```

Comment:
We could skip `setKeyPartitionList` (and `getPrivateKeyName`) for a temporary keychain we just created, but it's doesn't take particularly long (200-400ms). Also it's possible this step will be necessary for temporary keychains as well in a future OS release.

Reply:
I did all my task testing on High Sierra. I simulated set-key-partition-list not being present by typoing the name. :)

I tested security set-key-partition-list manually on Sierra and El Capitan (10.11). I could set up an agent on our Sierra mac mini if we don't want to wait for these changes to hit TFS ring 0.

Reply:
I updated the code to skip the `security set-key-partition-list` call for the temporary keychain.